manager/attestation_policy.go

```diff
@@ -67,11 +68,30 @@ func (ms *managerService) FetchAttestationPolicy(_ context.Context, computationI
 			return nil, err
 		}
 	case vmi.Config.EnableSEVSNP:
-		measurement, err = guest.CalcLaunchDigest(guest.SEV_SNP, vmi.Config.SMPCount, uint64(cpuid.CpuSigs[vmi.Config.CPU]), vmi.Config.OVMFCodeConfig.File, vmi.Config.KernelFile, vmi.Config.RootFsFile, strconv.Quote(qemu.KernelCommandLine), defGuestFeatures, "", vmmtypes.QEMU, false, "", 0)
+		args := []string{}
```

Comment:
use the https://github.com/ultravioletrs/cocos/blob/4bb732ebf9c44b1c8d9e365a3b6c34f0f74ef1d6/pkg/attestation/igvmmeasure/igvmmeasure.go#L25 to execute the measurement calculation command